attributes/default.rb

```diff
@@ -66,7 +66,18 @@
       '/media/downloadable/',
       '/pkginfo/',
       '/report/config.xml',
-      '/var/'
+      '/var/',
+      '/.gitignore',
+      '/.htaccess',
+      '/.htaccess.sample',
+      '/composer.json',
+      '/cron.php',
+      '/cron.sh',
+      '/index.php.sample',
+      '/install.php',
+      '/mage',
```

Comment:
fair enough, I'd looked and didn't see it there, guess I should do with an eye check :)